src/collectors/darner/darner.py

```diff
@@ -137,7 +137,7 @@ def collect(self):
                             queues[queue][queue_stat])
 
             # figure out what we're configured to get, defaulting to everything
-            desired = self.config.get('publish', stats.keys())
+            desired = self.config.get('publish', list(stats.keys()))
```

Comment:
Sorry, I mean, normalize on what six does. They provide a wrapper to do the same behavior in both.